### ISSUE TITLE
Incorrect Name

### ISSUE THREAD

**Issue by b4ldr** [RIPE-NCC/ripe.atlas.sagan#50]
there is a property of nameserver_count for nscount at 
https://github.com/RIPE-NCC/ripe.atlas.sagan/blob/master/ripe/atlas/sagan/dns.py#L57-L60

however this is actually the authoritative section count

**Comment by danielquinn** [comment]
I'm a little confused.  Are you saying that `NSCOUNT` as we're getting from the abuf parser is actually the authoritative section count, or that something in the `Header` class is amiss we just need to rename `nscount`?

**Comment by b4ldr** [comment]
Well its possible its not a bug the rfc says

```
NSCOUNT         an unsigned 16 bit integer specifying the number of name
                server resource records in the authority records
                section.
```

However the authority section can contain more then just name server records e.g. dig mnasd.google.com will have an soa in the auth section.  So, yes, nscount is actually the count of authority sections and is very badly named, double check with anand to make sure im not misinterpreting things but thats my understanding

**Comment by danielquinn** [comment]
Alright.  I'll have a chat with Philip and Anand once the RIPE meeting is over and we'll see what can be done about this.

**Comment by danielquinn** [comment]
Alright, I just had a talk with Philip and he can't reproduce the problem.  Do you have an example result that we can poke at and reproduce?

**Comment by b4ldr** [comment]
Its not an error that can be reproduced.  I just think that nameserver_count would be better renamed as authority_count.  however its a really minor thing so feel free to close this

**Comment by danielquinn** [comment]
Alright, so I just talked with Philip on this one, and I think we're gonig to keep the name as-is, if for no other reason that it best reflects the original name `NSCOUNT`.  I considered setting up an alias called `authority_count`, but that would introduce two things that do the same thing for the sake of naming, which doesn't seem cool.

I did however just add a docstring to the nameserver_count for someone who might be poking around the source, and a bit more explanation in the documentation, to hopefully make things a little more obvious.
